spacy_llm/backends/rest/backend/base.py

```diff
@@ -38,7 +42,8 @@ def __init__(
         self._check_api_endpoint_compatibility()
 
         assert self._max_tries >= 1
-        assert self._timeout >= 1
+        assert self._interval >= 1
```

Comment:
That seems a bit over-constrained? We should at least document it in the docstrings?

Reply:
Ah those are integers... Is there a particular why? I think users would expects floats there

Reply:
True, done in https://github.com/explosion/spacy-llm/pull/110/commits/0e8a3ca68d22ad96f9e1be5a8873d72c07cd7a9e.